frontend/src/components/activePage/TableInstance/TableInstanceLogic.tsx

```diff
@@ -51,13 +55,16 @@ const TableInstanceLogic: FC<ITableInstanceLogicProps> = ({ ...props }) => {
           if (!data?.updateInstance?.instance) return prev;
 
           const { instance, updateType } = data?.updateInstance;
+          let isPrettyNameUpdate;
```

Comment:
```suggestion
          let isPrettyNameUpdate = false;
```
also in TableWorkspaceLogic